src/sentry/static/sentry/app/views/settings/incidentRules/metricField.tsx

```diff
@@ -41,6 +42,12 @@ const getFieldOptionConfig = (dataset: Dataset) => {
         return ['tags[sentry:user]', 'string'];
       }
 
+      if (organization.features.includes('measurements')) {
+        if (key.startsWith('measurements.')) {
+          return [key, measurementType(key)];
+        }
+      }
```

Comment:
Do you also need to remove measurement fields if the organization doesn't have measures? It looks like line 51 will return the measure with undefined as a type.